packages/components/addon/components/hds/form/helper-text/index.hbs

```diff
@@ -0,0 +1,3 @@
+<div class={{this.classNames}} id={{this.id}} ...attributes>
```

Comment:
[question] Is there a reason this doesn't have a pre-defined class name?

Reply:
can you expand on this? I am not sure of the question

Reply:
Let me try this another way: Is there a specific reason that the helper text has `this.classNames` but the error message has `hds-form-error__message`?